ibis/backends/duckdb/tests/conftest.py

```diff
@@ -2,13 +2,54 @@
 from pathlib import Path
 
 import ibis
+from ibis.backends.conftest import TEST_TABLES
 from ibis.backends.tests.base import BackendTest, RoundAwayFromZero
 
 
 class TestConf(BackendTest, RoundAwayFromZero):
     def __init__(self, data_directory: Path) -> None:
         self.connection = self.connect(data_directory)
 
+    @staticmethod
+    def load_data(data_dir, script_dir, **kwargs):
+        """Load testdata into an impala backend.
+
+        Parameters
+        ----------
+        data_dir : Path
+            Location of testdata
+        script_dir : Path
+            Location of scripts defining schemas
+        """
+
+        database = (
+            "ibis_testing"
+            if not kwargs.get("database", None)
+            else kwargs["database"]
+        )
+
+        schema = (
+            open(script_dir / 'schema' / 'duckdb.sql')
+            if not kwargs.get("schema", None)
+            else kwargs["schema"]
+        )
+        tables = (
```

Comment:
If we don't define table schemas anywhere else then baked in `TEST_TABLES` would be enough here as well.